### ISSUE TITLE
Docker failures in Vagrant due to missing /etc/sysconfig/mantl-storage

### ISSUE THREAD

**Issue by siddharthist** [CiscoCloud/mantl#1403]
As @stevendborrelli pointed out on #1365, Vagrant builds are failing:

> ```
TASK: [docker | enable docker] ************************************************ 
failed: [control-01] => {"failed": true}
msg: Job for docker.service failed because a configured resource limit was exceeded. See "systemctl status docker.service" and "journalctl -xe" for details.
```
But this is due to the new docker implementation not creating a `/etc/sysconfig/mantl-storage` file on non-lvm based systems and not a problem with this PR. 

- Ansible version (`ansible --version`): n/a - 1.9 in VM 
- Python version (`python --version`): n/a
- Git commit hash or branch: master
- Cloud Environment: Virtualbox/Vagrant
- Terraform version (`terraform version`): n/a

**Comment by siddharthist** [comment]
@stevendborrelli Do you know what went in that file? I'm having trouble finding it in the git history.

**Comment by siddharthist** [comment]
Closed in #1410
